drydock/plugin.py

```diff
@@ -1,11 +1,56 @@
 from glob import glob
 import os
+import yaml
 import pkg_resources
 
+from tutor import env as tutor_env
+from tutor import config as tutor_config
 from tutor import hooks
 
+import logging
+
 from .__about__ import __version__
 
+INSTALLATION_PATH = f'{os.getcwd()}/env'
+
+INIT_JOBS_SYNC_WAVE = -100
+
+
+def get_init_tasks():
+    """Return the list of init tasks to run."""
+    init_tasks = list(hooks.Filters.CLI_DO_INIT_TASKS.iterate())
+    config = tutor_config.load(os.getcwd())
+
+    # Standarize deprecated COMMANDS_INIT Filter, to be removed in palm or later
+    standarized_commands = []
+    for service, init_path in list(hooks.Filters.COMMANDS_INIT.iterate()):
+        standarized_commands.append((service, tutor_env.read_template_file(*init_path)))
+
+    init_tasks.extend(standarized_commands)
+
+    templates_path = f'{INSTALLATION_PATH}/k8s/jobs.yml'
+    templates = list(yaml.load_all(open(templates_path, 'r', encoding='utf-8'), Loader=yaml.FullLoader))
+
+    response = []
+    for i, (service, definition) in enumerate(init_tasks):
+        for template in templates:
+            if template['metadata']['name'] == service + '-job':
+                render_definition = tutor_env.render_str(config, definition)
+                data = dict()
+                container = template['spec']['template']['spec']['containers'][0]
+                data['volumes'] = template['spec']['template']['spec']['volumes'] if 'volumes' in template['spec']['template']['spec'] else []
+                data['name'] = service
+                data['command'] = render_definition
```

Comment:
The Kubernetes runner in tutor performs a [validation](https://github.com/overhangio/tutor/blob/v15.3.7/tutor/commands/k8s.py#L103-L112) to define the command in the job. I think we should do something similar just in case. It applies to Palm and Quince as well

Reply:
Is not necessary because we're not using all definition of the job templates, we're defining literal the [command](https://github.com/eduNEXT/drydock/pull/79/files#diff-a796dc55854880fca04c5214fd9ac6f2adbe291f87b50c2ef0d74be0ef48b04bR22-R25) key